sanic_openapi/swagger.py

```diff
@@ -30,6 +30,12 @@ def remove_nulls(dictionary, deep=True):
     }
 
 
+def get_swagger_url(app):
+    bp = app.blueprints.get('swagger')
```

Comment:
A question might be a little out of scope. Since that we allow the user to change `swagger_blueprint.prefix_url`, how to prevent user also change `swagger_blueprint.name` and make sure your code will always work?